azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -61,24 +62,26 @@ def create_from_connection_string(cls, connection_string, ca_cert=None):
         # in order to differentiate types of connection strings.
         authentication_provider = auth.SymmetricKeyAuthenticationProvider.parse(connection_string)
         authentication_provider.ca_cert = ca_cert  # TODO: make this part of the instantiation
-        iothub_pipeline = pipeline.IoTHubPipeline(authentication_provider)
+        iothub_pipeline = pipeline.IoTHubPipeline(authentication_provider, **kwargs)
```

Comment:
>**kwargs [](start = 75, length = 8)

I actually think options should be instantiated in this scope rather than passing the kwargs down. I think we should convert kwargs into a concrete object as soon as possible.